lib/src/command/global_activate.dart

```diff
@@ -38,6 +38,11 @@ class GlobalActivateCommand extends PubCommand {
     argParser.addFlag("overwrite",
         negatable: false,
         help: "Overwrite executables from other packages with the same name.");
+
+    argParser.addOption("source-url",
```

Comment:
`--server-url` or `--hosted-url` might be better, the key for this in the pubspec is `dependencies.<packageName>.hosted.url`

Or maybe `--hosted-source-url`

Reply:
+1 on `--hosted-url`